content/blog/2020/06/2020-07-07-custom-distribution-service.adoc

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "Custom Distribution Service : Midterm Summary"
+tags:
+- service
+- distribution
+- cloud-native
+- gsoc
+- gsoc2020
+- packaging
+- platform-sig
+author: sladyn98
+opengraph:
+  image: /images/gsoc/cds_gsoc.png
```

Comment:
Would be better to put it somewhere in `post-images` or, if you plan to reuse it, in `/images/gsoc/2020/custom-distribution-service/opengraph.png`

Reply:
Sounds like a good idea :)

Reply:
`video::966OiFW2gow[youtube]`